test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -383,6 +383,95 @@ public async Task GivenAResource_WhenNotAuthorized_ThenAuditLogEntriesShouldBeCr
                 expectedAppId: TestApplications.NativeClient.ClientId);
         }
 
+        [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        [Trait(Traits.Category, Categories.Batch)]
+        public async Task GivenABatch_WhenPost_ThenAuditLogEntriesShouldBeCreated()
```

Comment:
I thought there was a reason why the test was split into R4 and Stu3 because there was some difference between the two?

Reply:
Oh I see, it's the http status code that's different?

Reply:
Yea, once I fixed the differences between the between the sample batches the only differences were the status codes in the case of an `IfMatch` failure.